openmetadata-ui/src/main/resources/ui/src/components/Task/TaskTab/TaskTab.component.tsx

```diff
@@ -276,23 +274,25 @@ export const TaskTab = ({
         {getTaskLinkElement}
       </Col>
       <Col span={24}>
-        <Typography.Text className="text-grey-muted">
-          {t('label.assignee-plural')}:{' '}
-        </Typography.Text>
-
-        <OwnerLabel
-          hasPermission={false}
-          owner={taskDetails?.assignees[0]}
-          onUpdate={noop}
-        />
-        <Typography.Text className="text-grey-muted">
-          {t('label.created-by')}:{' '}
-        </Typography.Text>
-        <OwnerLabel
-          hasPermission={false}
-          owner={{ name: task.createdBy, type: 'user', id: '' }}
-          onUpdate={noop}
-        />
+        <div className="d-flex items-center gap-2">
+          <Typography.Text className="text-grey-muted">
+            {t('label.assignee-plural')}:{' '}
+          </Typography.Text>
+
+          <OwnerLabel
+            hasPermission={false}
+            owner={taskDetails?.assignees[0]}
+            onUpdate={noop}
+          />
+          <Typography.Text className="text-grey-muted">
```

Comment:
@karanh37 we need to add gap between Assignee and Current owner
You have added gap between Assignee label / Assignee value / Created by Label / Created by value